src/kanban/kanban.js

```diff
@@ -0,0 +1,28 @@
+let isDropdownOpened;
```

Comment:
лучше сразу явно присвоить дефолтное значение.
```suggestion
let isDropdownOpened = false;
```